static/js/zamboni/devhub.js

```diff
@@ -188,6 +188,26 @@ $(document).ready(function() {
             $modalUnlist.render();
         }
     }
+    $(document.body).delegate('[data-confirm]', 'submit', function(e) {
```

Comment:
Could you please add a comment here, explaining what this does? Eg "display the confirmation modal on form submission."